src/app/components/kdv-select/kdv-select.component.ts

```diff
@@ -30,6 +30,7 @@ export class KdvSelectComponent implements OnInit, OnDestroy {
   @Input() useDescription: boolean;
   @Input() filter: (number) => boolean;
   @Input() getIconFunc: (kdvElement: KdvElement) => string;
+  @Input() obsLocMode: boolean = false;
```

Comment:
Er det verdt den ekstra kompleksiteten å ha en egen modus for dette?
Har du snakket med Atle om at det er spesielle krav til dropdown som brukes til sted og tid?
Mulig han ikke har vært bevisst på at dropdown det ser annerledes ut i skissene enn andre steder.

Reply:
Han er bevisst, og mente at det var verdt å gjøre spesielle tilpassinger for dette tilfellet.